draftivist_ui/src/components/DraftClosing.tsx

```diff
@@ -0,0 +1,60 @@
+import * as m from "mithril";
+import { Campaign, Issue } from "../models"
+import { BaseComponent, Link, elementAttrs } from "./base"
+import { PreviewDraftCTA } from './PreviewDraftCTA';
+
+type Attrs = {
+  campaign: Campaign
+  issue: Issue
+}
+
+export default function (): BaseComponent<Attrs> {
+
+  function saveDraft(campaign: Campaign, content: string) {
+    campaign.saveToDraft('conclusion', content);
+  }
+
+  let inputValue = '';
+
+  return {
+    ...elementAttrs,
+    oninit: (vnode) => {
+      if (vnode.attrs.campaign.draft.conclusion) {
+        inputValue = vnode.attrs.campaign.draft.conclusion;
+      }
+    },
+    view: (vnode) => {
+      return (
+        <div className='draft_body'>
+          <div className="draft_section_heading">
```

Comment:
I think you mentioned that the css wasn't done yet, so ignore me if you already know this, but the grid won't display properly unless all the children have defined column spans.